README.md

```diff
@@ -58,6 +58,30 @@ With Parse SDK 2.0.0, gone are the backbone style callbacks and Parse.Promises.
 
 We have curated a [migration guide](2.0.0.md) that should help you migrate your code.
 
+## Want to ride the bleeding edge?
+
+It is recommend to use builds deployed npm for many reasons, but if you want to use
```

Comment:
this first sentence needs some work.

"We recommend using the most recent tagged build published to npm for production.  However, you can test not-yet-released versions of the Parse-SDK-JS by referencing specific branches in your `package.json`.  For example, to use the master branch: ...."